bazel/unix_cc_configure.bzl

```diff
@@ -158,10 +182,31 @@ def _is_gold_supported(repository_ctx, cc):
   ])
   return result.return_code == 0
 
+def _get_no_canonical_prefixes_opt(repository_ctx, cc):
+  # If the compiler sometimes rewrites paths in the .d files without symlinks
+  # (ie when they're shorter), it confuses Bazel's logic for verifying all
+  # #included header files are listed as inputs to the action.
+
+  # The '-fno-canonical-system-headers' should be enough, but clang does not
+  # support it, so we also try '-no-canonical-prefixes' if first option does
+  # not work.
+  opt = _add_option_if_supported(repository_ctx, cc,
+                                 "-fno-canonical-system-headers")
```

Comment:
does this fit on one line?  What line-length limit are we going for in this file?  Here & 2 lines below.